frontend/packages/ux-editor/src/components/FormComponent.tsx

```diff
@@ -118,7 +119,7 @@ export const FormComponent = memo(function FormComponent({
             />
           ) : (
             <div className={classes.formComponentTitle}>
-              <i className={formItemConfigs?.[component.type]?.icon || 'fa fa-help-circle'} />
+               <span className={classes.icon} >{Icon  && <Icon aria-label={"icon"}/>} </span>
```

Comment:
Tittelen på ikonet bør være mer beskrivende enn dette. Bruk f.eks. `getComponentTitleByComponentType(component.type, t)`. Ikonene skal kunne ta imot et `title`-attributt, så du kan bruke det i stedet for `aria-label`. (`aria-label` er for situasjoner hvor ingen andre måter å navngi komponenten på fungerer.) Hvis du får til å sette typen slik jeg foreslo i en annen kommentar, bør du ikke få typefeil på det.

Reply:
Jeg brukt title={"icon"} isteden for getComponentTitleByComponentType(component.type, t) fordi den er brukt allerede  i samme <diven   og når jeg brukte den, mange tester feilet med feil som sier at det er flere elementer i samme navn.